### ISSUE TITLE
DwC-A: cannot import occurrence data for existing species 

### ISSUE THREAD

**Issue by sergeitarasov** [SpeciesFileGroup/taxonworks#2581]
As an experiment, I tried to import (many times with different modifications) one occurrence record using DwC-A (attached) with ‘restrict record to existing nmcl’ to match the species that is already in TW. But it does not work:

`Protonym Parachorius not found with that name and/or classification. Importing new names is disabled by import settings.`

I wonder how can I fix it? :)

[DwC_Parach.xlsx](https://github.com/SpeciesFileGroup/taxonworks/files/7347083/DwC_Parach.xlsx)
?

**Comment by LocoDelAssembly** [comment]
@sergeitarasov Sorry late reply, had some problems with the database and long meeting afterwards.

I tried with the attached spreadsheet and it had no trouble handling the name, but complained that the repository referenced in `institutionCode` with acronym `AN` does not exist (I changed it to something else to test):
![image](https://user-images.githubusercontent.com/4991925/137384456-cc5880e6-e147-4d90-992c-09cc1cd33978.png)
![image](https://user-images.githubusercontent.com/4991925/137384493-831933a5-9d87-4e8e-9e3f-f728993df93b.png)

Spreadsheet used: [DwC_Parach.xlsx](https://github.com/SpeciesFileGroup/taxonworks/files/7348941/DwC_Parach.xlsx) (`higherClassification` added on the far right)

**Comment by LocoDelAssembly** [comment]
@sergeitarasov right, sorry. On first try I forgot to use existing nomenclature, so it created the duplicate, but on second try I enabled and still succeeded. Investigating why is happening...

![image](https://user-images.githubusercontent.com/4991925/137397930-c543da2a-facd-42f2-8693-4cdd238848ed.png)

**Comment by LocoDelAssembly** [comment]
Found the problem. The importer was expecting authorship and year as data in the taxon name itself (as that is the way it creates them), but was failing in this case because authorship is derived from original citation. Fixed by matching by rank and name only, now it matched existing name:
![image](https://user-images.githubusercontent.com/4991925/137402611-d58ac779-6a47-4ce9-b22a-3d75adfd6092.png)
![image](https://user-images.githubusercontent.com/4991925/137402626-0671b139-1127-4f5f-9787-5d9999bdbc19.png)
![image](https://user-images.githubusercontent.com/4991925/137402639-f9893b96-cf77-4723-aa9c-5a75a0ecd9b1.png)

**Comment by LocoDelAssembly** [comment]
@sergeitarasov forgot to mention, this fix won't be available until we release 0.20.1. Please be sure to delete the duplicate name created by the importer to avoid confusion.

**Comment by sergeitarasov** [comment]
Works for me now! Thanks for the help @LocoDelAssembly  :)
Two quick Qs:

1. Does that mean that the import for taxon name linked to author citation vs. linked to verbatim authroship is different at the moment?
2. When approximately  TW 0.20.1 will be realesed?

**Comment by mjy** [comment]
Can we close this?
